docs/products/kafka/howto/use-kafka-topic-catalog.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Manage Apache Kafka topics with topic catalog
+limited: true
+---
+
+The Aiven for Apache Kafka® topic catalog offers a user-friendly interface to manage your Apache Kafka topics within your Aiven for Apache Kafka services.
+
+## Access topic catalog
+
+1. Log in to the [Aiven console](https://console.aiven.io/).
+1. Click **Tools** in the top menu.
+1. Select **Kafka topic catalog**.
+
+## Browse the topic catalog
+
+On the Kafka topic catalog page, you can:
+
+- **Table view**: This default view lists all topics in a table format, showing columns
+  for the topic name, service, project, and owner.
+- **Card view**: Click **Tiles** to switch to the card view and browse topics displayed
+  as cards.
+- **Search for Apache Kafka topics**: Enter the topic name in the search bar at the top
+  to find specific topics.
+- **Filter Kafka topics**: Use the filter options to narrow down topics by:
+
+  - **Project**: Display topics for a specific project.
+  - **Service**:  Display topics for a specific Aiven for Apache Kafka service
+  - **Owner group**:  Display topics by ownership group when governance is
+    enabled for your organization.
```

Comment:
I'm still working on this filtering and it will most likely come shortly after LA, due to targeting smaller bugs.

Not sure if we should leave it in on the assumption the filtering is coming or not though.